src/components/tutorial-meta/components/variant-list/index.tsx

```diff
@@ -0,0 +1,57 @@
+// @TODO uncomment when enabling cookies
+// import Cookies from 'js-cookie'
+import Link from 'components/link'
+import classNames from 'classnames'
+import {
+	TutorialVariant,
+	TutorialVariantOption,
+	getVariantParam,
+	getVariantPath,
+} from 'views/tutorial-view/utils/variants'
+import { useRouter } from 'next/router'
+import s from './variant-list.module.css'
+
+export default function VariantList({ variant }: { variant: TutorialVariant }) {
+	// @TODO hook this into a useVariants hook
+	const activeVariantOption = variant.options[0]
+	const VARIANT_LIST_ID = 'variant-list-label'
+	const { asPath } = useRouter()
+
+	return (
+		<div className={s.root}>
+			<label id={VARIANT_LIST_ID} className={s.label}>
+				{variant.name}
+			</label>
+			<nav>
+				<ul aria-labelledby={VARIANT_LIST_ID} className={s.list}>
+					{variant.options.map((option: TutorialVariantOption) => {
+						const variantParam = getVariantParam(variant.slug, option.slug)
+						const isActiveOption = activeVariantOption.slug === option.slug
+						return (
+							<li key={option.slug}>
+								<Link
+									className={classNames(
+										s.link,
+										isActiveOption ? s.activeLink : s.inActiveLink
```

Comment:
🙋🏻 Thoughts on hooking the `activeLink` styles directly into the state of the link via `[aria-current="page"]` selector? Have done this in the past in a few components to ease maintenance of accessibility requirements, inspired by [Using CSS to Enforce Accessibility](https://adrianroselli.com/2021/06/using-css-to-enforce-accessibility.html).

Reply:
That's a good idea!

Reply:
Refactored in https://github.com/hashicorp/dev-portal/pull/1931/commits/04030ec10beb11bc119b572409a77f31b06a5a9d 💖